lib/openh264.py

```diff
@@ -41,9 +41,10 @@ def EncodeCommandLine(self, parameters, bitrate, videofile, encodedfile):
         '-bf %s '
         '-sw %d -sh %d '
         '-dw 0 %d -dh 0 %d '
-        '-rc 0 -maxbrTotal %d -tarb %d '
+        '-rc 0 -maxbrTotal 0 -tarb %d '
```

Comment:
Hi yes, the default values in branch1.4 of maxbr are not 0 in cfgs. If do not use the command line to replace the default values in cfgs, it will take the value in cfgs. If using a cfg where the values of max-brs in cfgs are all 0, we can remove this option. I am not sure if changing the "compile_tools.sh" to modify cfg directly as you did with FrameSkip is better.
There is a planned pull request to change the default value of max-brs in cfg to openh264-master.